enterprise/forms.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+Forms to be used in the enterprise djangoapp
+"""
+from django import forms
+from enterprise.models import EnterpriseCustomer
+from enterprise.utils import get_all_catalogs
+
```

Comment:
@zubair-arbi same code quality comment. Also, I believe it's dictated by `pep8`, not consistency, :)